scripts/osfstorage/settings/local-dist.py

```diff
@@ -0,0 +1,15 @@
+# encoding: utf-8
+
+USERNAME = '...'
```

Comment:
For unset values, better to use `None` in case we ever need to do something like `if settings.USERNAME`.